pontoon/base/static/js/translate.js

```diff
@@ -429,7 +429,21 @@ var Pontoon = (function (my) {
      * Update current translation length
      */
     updateCurrentTranslationLength: function () {
-      $('#translation-length .current-length').html($('#translation').val().length);
+      var limit = this.translationLengthLimit,
+          translation = $('#translation').val();
+
+      if (limit) {
+        var length = this.stripHTML(translation).length,
```

Comment:
I started to wonder how should We handle various placeables like e.g. `%s` or `{0}` because they can change the length of result string.